controllers/utils_test.go

```diff
@@ -18,48 +18,44 @@ var _ = Describe("Controller Suite", func() {
 		BeforeEach(func() {
 			master = &v1alpha1.CDAPMaster{}
 			serviceToSpec = map[string]*v1alpha1.CDAPServiceSpec{
-				serviceLogs:                &master.Spec.Logs.CDAPServiceSpec,
-				serviceAppFabric:           &master.Spec.AppFabric.CDAPServiceSpec,
-				serviceAppFabricProcessor:  &master.Spec.AppFabricProcessor.CDAPServiceSpec,
-				serviceMetrics:             &master.Spec.Metrics.CDAPServiceSpec,
-				serviceRouter:              &master.Spec.Router.CDAPServiceSpec,
-				serviceMessaging:           &master.Spec.Messaging.CDAPServiceSpec,
```

Comment:
Keep this. Messaging service is not permanently disabled. It can be optionally disabled if needed.